redbeams-api/src/main/java/com/sequenceiq/redbeams/api/endpoint/v4/database/request/CreateDatabaseV4Request.java

```diff
@@ -0,0 +1,79 @@
+package com.sequenceiq.redbeams.api.endpoint.v4.database.request;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.sequenceiq.cloudbreak.api.endpoint.v4.JsonEntity;
+import com.sequenceiq.cloudbreak.doc.ModelDescriptions.Database;
+
+import io.swagger.annotations.ApiModel;
+import io.swagger.annotations.ApiModelProperty;
+
+/**
+ * A request for creating a database on a database server.
+ */
+@ApiModel
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class CreateDatabaseV4Request implements JsonEntity {
+
+    @ApiModelProperty(Database.NAME)
+    private String existingDatabaseServerName;
```

Comment:
Maybe this should be the CRN of the database server. Maybe environment should be included too. I just think that name alone won't be specific enough.

Reply:
Yeah, thanks for catching all of these -- I was a little unclear if we're doing workspaces or environments or what.